packages/scanner/src/page.ts

```diff
@@ -60,7 +60,14 @@ export class Page {
         const axePuppeteer: AxePuppeteer = await this.axePuppeteerFactory.createAxePuppeteer(this.puppeteerPage);
         const scanResults = await axePuppeteer.analyze();
 
-        return { results: scanResults };
+        if (response.request().redirectChain().length > 0) {
```

Comment:
General question : Is there any value in finding out the type of redirect? Some redirects are temporary, some are permanent. Does the WCP team care about this? 
Additionally is the WCP team interested in the number of redirects ? (I think they shouldn't care)